packages/@o3r/localization/builders/metadata-check/schema.json

```diff
@@ -0,0 +1,46 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "type": "object",
+  "$id": "LocalizationMigrationMetadataCheckBuilderSchema",
+  "title": "Check localization migration metadata builder",
+  "description": "",
+  "properties": {
+    "migrationMetadataFolder": {
+      "type": "string",
+      "description": "Path to the file containing the migration metadata."
+    },
+    "granularity": {
+      "type": "string",
+      "description": "Granularity of the migration check.",
+      "default": "minor"
+    },
+    "allowBreakingChanges": {
+      "type": "boolean",
+      "description": "Are breaking changes allowed.",
+      "default": true
+    },
+    "packageManager": {
+      "type": "string",
+      "description": "Override of the package manager, otherwise it will be determined from the project."
+    },
+    "metadataPath": {
+      "type": "string",
+      "description": "Path of the localization metadata file.",
+      "default": "./localization.metadata.json"
```

Comment:
I think per default the file name should be `./localisation.metadata.json`
https://github.com/AmadeusITGroup/otter/blob/main/packages/%40o3r/localization/builders/localization-extractor/schema.json#L37